test/unit/ngsiv2/HTTP_reveice_measures-test.js

```diff
@@ -79,13 +79,11 @@ describe('HTTP: Measure reception ', function() {
 
         nock.cleanAll();
 
-        // Note /v1/updateContext response is not processed by IOTA so its content is irrelevant,
-        // as far as it is a 200 OK
         contextBrokerMock = nock('http://192.168.1.1:1026')
             .matchHeader('fiware-service', 'smartGondor')
             .matchHeader('fiware-servicepath', '/gardens')
-            .post('/v1/updateContext')
-            .reply(200, '{}');
+            .post('/v2/entities')
+            .reply(201);
```

Comment:
The right code is 204 (see http://telefonicaid.github.io/fiware-orion/api/v2/stable/)

It is not important (I guess that the IOTA agent looks for anything between 200 and 299) but better to use the right one in the tests ;)

Reply:
I have checked the url and it says: 
> Successful operation uses 201 Created. Reponse includes a Location header with the URL of the created entity.

This mock intercepts the request sent to the CB to create the entity which corresponds to the device. Am I missing something?

Reply:
You're fully right. Sorry the noise.

My mental process was /v1/updateContext -> (equivalent op in NGSIv2) -> /v2/op/update -> (response of that operation) -> 204. But of course this is a wrong deduction ;)

Reply:
NTC

Reply:
Don't worry at all, this kind of things happen to me all the time! xD